### ISSUE TITLE
Question about experimental results on math tasks

### ISSUE THREAD

**Issue by group1smile** [RUCAIBox/ALLO#2]
Awesome work! I reproduced SFT LLM on math tasks and got comparable results on GSM8k, MATH, and MAWPS. However, I only obtained 42.9 accuracy on TabMWP (56.7 in your paper). Could you please specify the training parameters including the learning rate, epoch, batch_size, etc?

**Comment by Timothy023** [comment]
Thank you for your attention to our work, and sorry for late reply.

There might be some details in the evaluation of TabMWP. In the parsing of LLM generated answer, if the ground truth answer is a number, we will extract the last number in the generated answer as the prediction of LLM.

Hope my response can solve your questions.
